pkg/summarizer/flakiness.go

```diff
@@ -0,0 +1,286 @@
+package summarizer
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/testgrid/internal/result"
+	"github.com/GoogleCloudPlatform/testgrid/pb/state"
+	"github.com/sirupsen/logrus"
+	"vbom.ml/util/sortorder"
+)
+
+var INFRA_REGEX = regexp.MustCompile(`^\w+$`)
```

Comment:
for syntax, this probably should be written as

```
var (
  INFRA_REGEX = ...
  JAILED_REGEX = ...
  DEFAULT_FLAKINESS = ...
  ...
)

Reply:
Done